### ISSUE TITLE
Internationalization / Translations

### ISSUE THREAD

**Issue by bumi** [getAlby/lightning-browser-extension#600]
In order to make it easier for people we should localize the app and provide translations. 
Opening this issue to track how we can approach this.

**Comment by escapedcat** [comment]
[raspiblitz-web](https://github.com/cstenglein/raspiblitz-web) ist running well with a [react-i18next](https://github.com/i18next/react-i18next) and [weblate](https://weblate.org/en/) setup:  
https://github.com/cstenglein/raspiblitz-web/tree/master/src/i18n

**Comment by escapedcat** [comment]
@pavanjoshi914 What setup would you suggest? You had a look at the raspiblitz-web project?

**Comment by pavanjoshi914** [comment]
> @pavanjoshi914 What setup would you suggest? You had a look at the raspiblitz-web project?

yeah, I just want to ask, if it is feasible to work on this at this moment. If yes, I will propose a detailed plan about, which setup we can integrate, locale management, translation key management , i18n rules for naming of keys etc

**Comment by escapedcat** [comment]
> yeah, I just want to ask, if it is feasible to work on this at this moment. If yes, I will propose a detailed plan about, which setup we can integrate, locale management, translation key management , i18n rules for naming of keys etc

Sounds great! I assume because this issue exists you're good to go ahead and propose a draft-setup for further discussions. Would that be ok?

**Comment by pavanjoshi914** [comment]
> > yeah, I just want to ask, if it is feasible to work on this at this moment. If yes, I will propose a detailed plan about, which setup we can integrate, locale management, translation key management , i18n rules for naming of keys etc
> 
> Sounds great! I assume because this issue exists you're good to go ahead and propose a draft-setup for further discussions. Would that be ok?

okay thanks!

**Comment by bumi** [comment]
I did not look into any of the options, yet. But you know it better anyway...
Suggestions and PRs on this are for sure welcome! thanks.

**Comment by pavanjoshi914** [comment]
sorry for the late! I had my semester exams so didn't have the bandwidth to work on this.
@bumi @escapedcat  here is the proposed i18n architecture for Alby
https://gist.github.com/pavanjoshi914/a65af473209541c3a5cfecbac0f6e70f

after a few revisions, this will also act as developer documentation for i18n in Alby! I will start implementation from now onwards. suggest any changes (if Any)

**Comment by escapedcat** [comment]
Setup was merged 🎉
